src/utils/getLanguageCode.ts

```diff
@@ -0,0 +1,13 @@
+import { LanguageCodeEnum } from 'react-helsinki-headless-cms';
+
+import type { Language } from '../types';
+
+const appLanguageToLanguageCode: Record<Language, LanguageCodeEnum> = {
+  en: LanguageCodeEnum.En,
+  fi: LanguageCodeEnum.Fi,
+  sv: LanguageCodeEnum.Sv,
+} as const;
```

Comment:
Does the `as const` do anything here? It does not change the compiled JS and it gives no advantage at least in VSCode. The way how you used the `as const satisfies Record` did change how it was interpreted as a readonly.

Reply:
Hmm, yes, it didn't do anything, changed it to "as const satisfies" so now it's more specific type.